src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -494,10 +494,10 @@
     <comment>Description for Web Request</comment>
   </data>
   <data name="WebRequestPortDataResultToolTip" xml:space="preserve">
-    <value>The result of the web request.</value>
+    <value>Content of a webpage as a string.</value>
```

Comment:
should it maybe be `Content of a web request as a string`?

Reply:
@martinstacey did you look at this one?

Reply:
addressed in commit d5247d3